docs/core/policies.rst

```diff
@@ -197,56 +197,59 @@ set the ``random_seed`` attribute of the ``KerasPolicy`` to any integer.
 Embedding Policy
 ^^^^^^^^^^^^^^^^
 
+    .. warning::
+
+        ``EmbeddingPolicy`` got renamed to ``TEDPolicy``. Please use :ref:`ted_policy` instead of ``EmbeddingPolicy``.
+        The functionality of the policy stayed the same.
+
+
+.. _ted_policy:
+
+TED Policy
+^^^^^^^^^^
+
 Transformer Embedding Dialogue Policy (TEDP)
```

Comment:
don't we just call it TED rather than TEDP?

Reply:
changed it to `Transformer Embedding Dialogue (TED) Policy`